src/parsers/i3d/PropertyLoader.ts

```diff
@@ -91,6 +92,8 @@ export default class PropertyLoader {
     'triangleCount':  (geometry: CGD) => { this.triangleCount                = geometry.indices.nextValue() ; },
     'thickness':      (geometry: CGD) => { this.thickness = this.values.radius![geometry.indices.nextValue()]; },
     'fileId':         (geometry: CGD) => { this.fileId    = this.values.fileId![geometry.indices.nextValue()]; },
+    'diagonalSize':   (geometry: CGD) => {
```

Comment:
What on Earth is a CGD? Why do we have a type named like this?

Reply:
CompressedGeometryData, defined at the top of the file. It's a long name and I wanted everything to line up nicely :)

Reply:
I disagree with this. Type names should be explicit and clear. CGD is confusing.

Reply:
Normally I'd agree with you, but this file is very repetitive, so I formatted it accordingly. Notice the //tslint: disable at the top

Reply:
Readability over everything

Reply:
I don't see why this makes it more readable. `CompressedGeometryData` is super easy to read. I agree with @nsunderland-cognite here.

Reply:
Talked to Nick/Anders, moved code around